### PR TITLE
Clean unit tests

### PR DESCRIPTION
**Pull request by Priya2698**
1. Clean `test_swizzle.cpp`.
2. Clean `test_scalar_hoisting.cpp` -- Add `UnaryOpType::sin` to evaluator.
3. Clean`test_no_op.cpp`

### CODE REVIEW

**Comment by Priya2698** [comment]
Merge after #1299. Some tests require `FullOp` and `IotaOp`.

File: test/test_scalar_hoisting.cpp [hunk 1]
```diff
@@ -397,7 +395,6 @@ __global__ void CUDAGeneratedKernel(int64_t i0, int64_t i1, int64_t i2, Tensor<i
       fusion.get(),
       cg_outputs,
       {start, end, step},
-      {t0, t1},
```
Comment (zasdfgbnm) [inline test/test_scalar_hoisting.cpp:400]: Should you remove the above code that computes `t0` and `t1`?

File: test/test_scalar_hoisting.cpp [hunk 2]
```diff
@@ -334,7 +332,7 @@ __global__ void CUDAGeneratedKernel(Tensor<float, 2, 2> T0, Tensor<float, 2, 2>
 
   assertCUDAKernel(fusion.get(), expected_kernel);
 
-  testValidate(fusion.get(), cg_outputs, {t0}, {t1}, __LINE__, __FILE__);
```
Comment (zasdfgbnm) [inline test/test_scalar_hoisting.cpp:337]: Should the computation of `t1` above be removed?

Comment (Priya2698) [reply test/test_scalar_hoisting.cpp:337]: Done. `clangtidy` does report unused variables but does not raise an error here.

**Review by zasdfgbnm — APPROVED** [review]

**Pull request merged**